src/compat/glibc_compat.cpp

```diff
@@ -67,6 +67,8 @@ __asm(".symver log2f_old,log2f@GLIBC_2.2.5");
 __asm(".symver log2f_old,log2f@GLIBC_2.4");
 #elif defined(__aarch64__)
 __asm(".symver log2f_old,log2f@GLIBC_2.17");
+#elif defined(__riscv)
```

Comment:
Is this necessary? We don't have to support much back-compat for riscv, as its glibc support is pretty new anyway.

Reply:
This might not be necessary. But we should drop `--enable-glibc-back-compat` flag when building for RISC-V

Reply:
I think he did this to appease the symbols check, the alternative would be to do a different symbols check per target platform.

Reply:
I did this so the compiler would know what `log2f_old` is.